README.md

```diff
@@ -19,4 +19,4 @@ Getting started
 
 * [Installation Guides](https://github.com/kitodo/kitodo-production/wiki/Installationsanleitung)
 * [User documentation](https://github.com/kitodo/kitodo-production/wiki/)
-* [Developer documentation](https://kitodo-production.readthedocs.io/en/latest/)
```

Comment:
What happens with the old documentation? Should it link to the new one?

Reply:
The old documentation was moved to the github wiki without changes, it is obsolete and will not be maintained any longer.